packages/forms/src/UIForm/fields/FieldTemplate/FieldTemplate.component.js

```diff
@@ -27,11 +27,27 @@ function FieldTemplate(props) {
 		[theme.updating]: props.valueIsUpdating,
 	});
 
+	const title = (
+		<div className={theme['field-template-title']}>
+			<Label id={props.id} label={props.label} />
+			{props.hint && (
+				<Action
```

Comment:
Why is this an `Action` ? This is not the right semantic IMO. What we need is not a button, but an icon, with a tooltip on focus/hover(/click ?)

Reply:
hmm, yes, indeed maybe better just icon & overlay trigger with onclick

as for event, I will clarify with UX, but I think most probably it will be click, at least for https://jira.talendforge.org/browse/TMC-16933